src/baskerville/blocks.py

```diff
@@ -1109,6 +1109,9 @@ def transformer(
     qkv_width=1,
     mha_initializer="he_normal",
     kernel_initializer="he_normal",
+    adapter=None,
+    latent=16,
```

Comment:
maybe combine into a single variable 'adapter_latent', which can still take default None?

Reply:
this is removed. now adapter insertion happens after model construction, and is defined in the json file.